### PR TITLE
Fully decompiled code_800D2DF0, code_80106760, code_801067A0, partially decompiled 80092320 (1 func…

### PR DESCRIPTION
**Pull request by NEstelami**
…tion remaining)

### CODE REVIEW

File: src/code_80092320.c.ignore [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#include <ultra64.h>
+#include <ultra64/os_internal.h>
```
Comment (camthesaxman) [inline src/code_80092320.c.ignore]: This header should only be included by code that is actually part of libultra.

File: src/code_801067A0.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include <ultra64.h>
+#include <global.h>
+
+// Note: This is part of libultra
+// TODO: CLEAN THIS UP!
+
+#define SP_STATUS_REG 0xA4040010
```
Comment (camthesaxman) [inline src/code_801067A0.c]: These SP register definitions should go into a new header ultra64/rsp.h. See https://github.com/camthesaxman/zelda_ocarina_mq_debug/blob/master/include/ultra64/rdp.h for how I'm defining them.

File: src/code_80106760.c [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#include <ultra64.h>
+#include <global.h>
+
+// Note: This is part of libultra
```
Comment (camthesaxman) [inline src/code_80106760.c]: Code that is part of libultra should go in the libultra subdirectory. There's a Makefile rule to compile everything in this subdirectory with correct `-O1` optimizations.

**Comment by camthesaxman** [comment]
Please change all the files to have LF line endings. The compiler gives a lot of warnings when CRLF is encountered.

**Comment by NEstelami** [comment]
Line endings have been fixed.

**Pull request merged**